pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -28,7 +32,7 @@ import (
 
 const (
 	// ControllerName is the name of the WMC controller
-	ControllerName = "windowsmachineconfig-controller"
+	ControllerName = "windowsMachineconfig-controller"
```

Comment:
Not sure why this change is required. Go naming standards need not apply to strings.

Reply:
Oops - replace error, will revert it